src/test/java/com/faforever/integration/factories/ClanFactory.java

```diff
@@ -0,0 +1,32 @@
+package com.faforever.integration.factories;
+
+import com.faforever.api.data.domain.Clan;
+import com.faforever.api.data.domain.Player;
+import lombok.Builder;
+
+public class ClanFactory {
+
+  public static Clan create() {
```

Comment:
I played around with the `@Builder` from lombok and if you must specify more you must call the `builder`